gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -115,6 +114,9 @@ export default function NxGlobalSidebarPage() {
             <NxCode>NxGlobalSidebarNavigation</NxCode> is a container for navigation links.
             It accepts all standard <NxCode>&lt;div&gt;</NxCode> HTML attributes.
           </NxP>
+          <NxP>
+            Note that it is strongly suggested that the number of navigation links used be kept to 10 or less.
```

Comment:
I think this should be stressed more strongly.  I'd make it a warning alert and use "must" language, not "strongly suggested" language.  I'd also add the reason why - that it's the most that can fit along with the footer at minimum page height.

Reply:
> I think this should be stressed more strongly. I'd make it a warning alert and use "must" language, not "strongly suggested" language. I'd also add the reason why - that it's the most that can fit along with the footer at minimum page height.

https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/1075ed3f27240a4c9f7f650619e8ef3be39fa421